store/src/java/com/zimbra/cs/datasource/DataSourceManager.java

```diff
@@ -220,7 +220,7 @@ public DataImport getDataImport(DataSource ds, boolean test) throws ServiceExcep
             } catch (Exception x) {
                 ZimbraLog.datasource.warn("Failed instantiating xsync class: %s", ds, x);
             }
-        case custom:
+        case oauth:
```

Comment:
There are references to `custom` in this case block which should be changed to `oauth`